Integrations/integration-CrowdStrikeFalcon.yml

```diff
@@ -0,0 +1,1095 @@
+        """
+        raw_entry = get_trasnformed_dict(behavior, DETECTIONS_BEHAVIORS_KEY_MAP)
+        raw_entry.update(extract_transformed_dict_with_split(behavior, DETECTIONS_BEHAVIORS_SPLIT_KEY_MAP))
+        return raw_entry
+
+
+    def resolve_detection(ids, status, assigned_to_uuid, show_in_ui):
+        """
+            Sends a resolve detection request
+            :param ids: Single or multiple ids in an array string format
+            :param status: New status of the detection
+            :param assigned_to_uuid: uuid to assign the detection to
+            :param show_in_ui: Boolean flag in string format (true/false)
+            :return: Resolve detection response json
+        """
+        payload = {
+            'ids': ids
+        }
+        if status:
+            payload['status'] = status
+        if assigned_to_uuid:
+            payload['assigned_to_uuid'] = assigned_to_uuid
+        if show_in_ui:
+            payload['show_in_ui'] = show_in_ui
+        # We do this so show_in_ui value won't contain ""
+        data = json.dumps(payload).replace('"show_in_ui": "false"', '"show_in_ui": false').replace('"show_in_ui": "true"', '"show_in_ui": true')
+        return http_request('PATCH', '/detects/entities/detects/v2', data=data)
+
+
+    def contain_host(ids):
+        """
+        UNTESTED - Contains host(s) with matching ids
+        :param ids: IDs of host to contain
+        :return: Contain host response json
+        """
+        payload = {
+            'ids': ids,
+            'action_name': 'contain'
+        }
+        data = json.dumps(payload)
+        return http_request('POST', '/devices/entities/devices-actions/v2', data=data)
+
+
+    def lift_host_containment(ids):
+        """
+        UNTESTED - Lifts off containment from host(s) with matchind ids
+        :param ids: IDs of host to lift off containment from
+        :return: Lift off containment response json
+        """
+        payload = {
+            'ids': ids,
+            'action_name': 'lift_containment'
+        }
+        data = json.dumps(payload)
+        return http_request('POST', '/devices/entities/devices-actions/v2', data=data)
+
+
+    ''' COMMANDS FUNCTIONS '''
+
+
+    def fetch_incidents():
+        """
+            Fetches incident using the detections API
+            :return: Fetched detections in incident format
+        """
+        last_run = demisto.getLastRun()
+        # Get the last fetch time, if exists
+        last_fetch = last_run.get('first_behavior_time')
+
+        # Handle first time fetch, fetch incidents retroactively
+        if last_fetch is None:
+            last_fetch, _ = parse_date_range(FETCH_TIME, date_format='%Y-%m-%dT%H:%M:%SZ')
+        last_fetch_timestamp = date_to_timestamp(last_fetch, date_format='%Y-%m-%dT%H:%M:%SZ')
+        fetch_query = demisto.params().get('fetch_query')
+        if fetch_query:
+            fetch_query = "first_behavior:>'{0}'+{1}".format(last_fetch, fetch_query)
```

Comment:
please use named place holders when formatting more than one variable in a string.